server/bundler/bin/bundler.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/env babel-node
```

Comment:
well, that's looking for babel-node in the shell's path. Do we want to use a babel-node that we install via a dev dependency? Or rely on the user having babel installed (and the right version) globally?